frontend/src/System/Status/MoreInfo/MoreInfo.js

```diff
@@ -12,52 +13,70 @@ class MoreInfo extends Component {
 
   render() {
     return (
-      <FieldSet legend="More Info">
+      <FieldSet legend={translate('MoreInfo')}>
         <DescriptionList>
-          <DescriptionListItemTitle>Home page</DescriptionListItemTitle>
+          <DescriptionListItemTitle>
+            {translate('HomePage')}
+          </DescriptionListItemTitle>
           <DescriptionListItemDescription>
             <Link to="https://sonarr.tv/">sonarr.tv</Link>
           </DescriptionListItemDescription>
 
-          <DescriptionListItemTitle>Wiki</DescriptionListItemTitle>
+          <DescriptionListItemTitle>
+            {translate('Wiki')}
+          </DescriptionListItemTitle>
           <DescriptionListItemDescription>
             <Link to="https://wiki.servarr.com/sonarr">wiki.servarr.com/sonarr</Link>
           </DescriptionListItemDescription>
 
-          <DescriptionListItemTitle>Forums</DescriptionListItemTitle>
+          <DescriptionListItemTitle>
+            {translate('Forums')}
+          </DescriptionListItemTitle>
           <DescriptionListItemDescription>
             <Link to="https://forums.sonarr.tv/">forums.sonarr.tv</Link>
           </DescriptionListItemDescription>
 
-          <DescriptionListItemTitle>Twitter</DescriptionListItemTitle>
+          <DescriptionListItemTitle>
+            {translate('Twitter')}
+          </DescriptionListItemTitle>
           <DescriptionListItemDescription>
             <Link to="https://twitter.com/sonarrtv">@sonarrtv</Link>
           </DescriptionListItemDescription>
 
-          <DescriptionListItemTitle>Discord</DescriptionListItemTitle>
+          <DescriptionListItemTitle>
+            {translate('Discord')}
+          </DescriptionListItemTitle>
           <DescriptionListItemDescription>
             <Link to="https://discord.sonarr.tv/">discord.sonarr.tv</Link>
           </DescriptionListItemDescription>
 
-          <DescriptionListItemTitle>IRC</DescriptionListItemTitle>
+          <DescriptionListItemTitle>
+            {translate('IRC')}
+          </DescriptionListItemTitle>
           <DescriptionListItemDescription>
             <Link to="irc://irc.libera.chat/#sonarr">#sonarr on Libera</Link>
```

Comment:
I think we could translate the link text as well

Reply:
added in #5823